book/src/semantics/name-resolving.md

```diff
@@ -0,0 +1,76 @@
+# Name resolving
```

Comment:
I wonder whether this should be in an "Internals" section. Maybe it's also helpful for users doing debugging?

(no need to resolve now tbc)

Reply:
Yea, this whole semantics chapter is meant for advanced users or maybe even for compiler team only.